src/renderer/tvgText.h

```diff
@@ -98,13 +98,13 @@ struct Text::Impl
     {
         if (!loader) return false;
 
+        loader->request(shape, utf8, fontSize, italic);
```

Comment:
I guess just only relocating this `request()` here from the 103 line will fix your problem. Could you please double-check it?

Reply:
That's right. I just made two steps too far. We can solve this issue this way. I will add another PR which makes cleaner why I decided to add `fontSize` to the `request()`.

Reply:
Got it. Could u please update the pr? We can fix the issue anyway.